src/CONST.ts

```diff
@@ -5325,6 +5325,22 @@ const CONST = {
             REPORT_ID: 'reportID',
             KEYWORD: 'keyword',
         },
+        SYNTAX_FILTER_VALUE_TO_KEY_MAPPING: {
+            date: 'DATE',
+            amount: 'AMOUNT',
+            expenseType: 'EXPENSE_TYPE',
+            currency: 'CURRENCY',
+            merchant: 'MERCHANT',
+            description: 'DESCRIPTION',
+            from: 'FROM',
+            to: 'TO',
+            category: 'CATEGORY',
+            tag: 'TAG',
+            taxRate: 'TAX_RATE',
+            cardID: 'CARD_ID',
+            reportID: 'REPORT_ID',
+            keyword: 'KEYWORD',
+        },
```

Comment:
I wonder if we could just re-use `SYNTAX_FILTER_KEYS` and manipulate the values, e.g. `toUpperCase()` etc